jenkins/run-compareSummary.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+INPUT_DATA_PATH="$1"
+RESULT_PATH="$2"
+BINPATH="$3"
+CASENAME="$4"
+FILENAME="$5"
+TOL="$6"
+OPM_OUTPUT_ROOT="$7"
+EXE_NAME="$8"
+TEST_ARGS="${@:9:100}"
+
+cd ${RESULT_PATH}
+${BINPATH}/${EXE_NAME} ${TEST_ARGS}
+cd ..
+
+#~/opm-output/build/bin/compareSummary ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/${CASENAME}/opm-simulation-reference/${FILENAME}
+
+${OPM_OUTPUT_ROOT}/build/bin/compareSummary ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/${CASENAME}/opm-simulation-reference/${FILENAME}
```

Comment:
this won't work. there's nothing mandating the use of a sibling-build for opm-output. see the patchi sent earlier this week  (or was it a branch link, don't recall)  where we set the path to this binary as a cmake variable in Findopm-output.cmake.